setup.py

```diff
@@ -54,14 +53,29 @@ def read_file(fname):
 static_requirements = []
 static_dir = os.path.dirname(os.path.realpath(kolibri_dist.__file__))
 
-install_requires = [str(ir.req) for ir in parse_requirements('./requirements.txt', session=False)]
+install_requires = [
+    'cherrypy==6.0.2',
+    'django-filter==0.13.0',
+    'django-js-reverse==0.7.2',
+    'six==1.10.0',
+    'djangorestframework==3.3.3',
+    'django==1.9.7',
+    'colorlog==2.6.0',
+    'docopt==0.6.2',
+    'django-mptt>=0.8.4',              # apparently you also need to have the dependency_links packages be included in install_requires
```

Comment:
In pip's case, when you add `--process-dependency-links`, it would pull in and read the `dependency_links` variable, and see if it matches the requirement you have in `install_requires`. Since the git version is 0.8.4, it satisfies our minimum django-mptt version and uses that.

If you don't include `--process-dependency-links`, then it doesn't read that variable and instead pulls everything from PyPI.